namespace_registry/client_commands_test.go

```diff
@@ -129,3 +136,57 @@ func TestServeNamespaceRegistry(t *testing.T) {
 	})
 	viper.Reset()
 }
+
+func TestRegistryKeyChaining(t *testing.T) {
+	viper.Reset()
+	// On by default, but just to make things explicit
+	viper.Set("Registry.RequireKeyChaining", true)
```

Comment:
If Registry.RequireKeyChaining is set to false in /etc/pelican/pelican.yaml, this test will use that rather than the viper.Set that occurs in on line. Somehow the pelican.yaml file is being read when it shouldn't be for this command.  Or being read to late so it's overwriting this line.